reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -1,35 +1,57 @@
 .heatmap {
   overflow-y: auto;
+  overflow-x: auto;
   max-height: 600px;
   height: 100%;
   background-color: #ffffff;
 }
 
-.heatmap table {
-  overflow-y: auto;
+.heatmap-container {
+  padding: 0;
 }
 
 .heatmap table th {
   vertical-align: top;
   position: sticky;
   top: 0;
   z-index: 2;
-  background-color: white;
+  background-color: inherit;
```

Comment:
This breaks the sticky header. Can't paste a screenshot, but try looking at the table after you've scrolled it down a bit.